autogluon/task/tabular_prediction/tabular_prediction.py

```diff
@@ -252,7 +315,25 @@ def fit(train_data, label, tuning_data=None, output_directory=None, problem_type
             set_best_to_refit_full : bool, default = False
                 If True, will set Trainer.best_model = Trainer.full_model_dict[Trainer.best_model]
                 This will change the default model that Predictor uses for prediction when model is not specified to the refit_full version of the model that previously exhibited the highest validation score.
-                Only valid if `refit_full=True`.
+                Only valid if `refit_full` is set.
+            save_bagged_folds : bool, default = True
+                If True, bagged models will save their fold models. This enables bagged models to be capable of inference after `fit()`.
```

Comment:
"fold models." -> 
"fold models (the models from each individual fold of bagging)."

"This enables bagged models to be capable of inference..." ->
"This is required to use bagged models for prediction..."

Reply:
Added